src/lib/command.ts

```diff
@@ -135,3 +135,17 @@ export async function sendPermissionsEmbed(msg: Message, error: string) {
         setTimeout(() => permMsg.delete(), PermissionsEmbedDelay); // Delete after 10 seconds
     }
 }
+
+export async function replyToCommandOrQuestion(msg: Message, embed: EmbedBuilder) : Promise<Message<boolean>> {
+    return msg.fetchReference()
+        .then((res) => {
+            let existingFooterText = '';
+            const existingFooter = embed.data.footer;
+            if (existingFooter) {
+                existingFooterText = `${existingFooter.text}\n\n`;
+            }
+            embed.setFooter({ text: `${existingFooterText} Executed by ${msg.author.tag} - ${msg.author.id}` });
+            return res.reply({ embeds: [embed] });
+        })
+        .catch(() => msg.reply({ embeds: [embed] }));
+}
```

Comment:
```suggestion
export async function replyToCommandOrQuestion(msg: Message, embed: EmbedBuilder) : Promise<Message<boolean>> {
    let existingFooterText = '';
    return msg.fetchReference()
        .then((res) => {
            const existingFooter = embed.data.footer;
            if (existingFooter) {
                existingFooterText = existingFooter.text;
            }
            embed.setFooter({ text: `${existingFooterText}\n\nExecuted by ${msg.author.tag} - ${msg.author.id}` });
            return res.reply({ embeds: [embed] });
        })
        .catch(() => msg.reply({ embeds: [embed] }));
}
```

This fixes the bug in my last testing and cleans up the text a bit as well (no unnecessary space)